lib/constants/signuplogin_constants.dart

```diff
@@ -0,0 +1,7 @@
+//Welcome in the build
+const String welcomeMessage = "Welcome to Aspire";
+//Account Verify
```

Comment:
Please leave an empty line to separate the two group of constants.

Reply:
Got it!